pipeline/stages/test.yaml

```diff
@@ -130,5 +130,63 @@ stages:
                 parameters:
                   subscriptionName: ${{ variables.subscriptionName }}
                   env: ${{ variables.env }}
-            
-            
\ No newline at end of file
+
+        - job: 'StopVM${{parameters.env}}'
```

Comment:
Can we convert this to a template in azure devops templates repo that accepts start/stop?

Reply:
Sure. Will add in.

Reply:
updated. variable TURN_OFF_VM_POST_RUN will allow this to be set.